source/wrangling.md

```diff
@@ -613,13 +441,11 @@ We use `.melt` to combine the Toronto, Montréal,
 Vancouver, Calgary, and Edmonton columns into a single column called `region`,
 and create a column called `mother_tongue` that contains the count of how many
 Canadians report each language as their mother tongue for each metropolitan
-area. We specify `value_vars` to be all
-the columns between Toronto and Edmonton:
+area:
 
 ```{code-cell} ipython3
 lang_mother_tidy = lang_wide.melt(
     id_vars=["category", "language"],
```

Comment:
This code block is different from the image above

Need to add a bold black comment for melissa to update the image (if the current code block is what we want to go with)

Reply:
I think it might be best to stick with the simpler code, so I will add a note

Reply:
Note added in 481